definitions/procedures/sync_plans.rb

```diff
@@ -0,0 +1,34 @@
+class Procedures::SyncPlans < ForemanMaintain::Procedure
```

Comment:
At the current state, it would probably make sense to have `SyncPlans::Enable` and `SyncPlans::Disable` procedures,
and have better message than `[enable/disable sync plans before/after upgrade]?, [y(yes), n(no), q(quit)]`.